### ISSUE TITLE
Various minor (obscure) UI bugs

### ISSUE THREAD

**Issue by tomcl** [tomcl/issie#275]
**Describe the bug**

1.  In some circumstances the "delete" icon and functionality does not appear on waveform names in waveform simulator. It should always appear when hovering on names.
2. Even when no change is made to design sheets, they need to be saved. This is probably due to the "auto-centre" logic which changes component positions. It should not do this unless components are a lot off centre. Schematic centering can be implemented through pan. Having to save (also this means sheets will be auto-saved on change) pollutes git repos with extra copies of sheets.
3. Schematic centering does not work when changing sheets - on balance it should.
4. In some circumstances right-clicking on a component does not make its properties come up. Clicking on sheet and back on component restores this functionality.

 - Version v3.0.11

**Additional context**
These things are not a great problem, but should be fixed!

**Comment by tomcl** [comment]
2. above is fixed in  https://github.com/tomcl/issie/commit/e20c3df699582a3c742f1f0fdd9df1f79d00de15

This will be merged into master when team2-wire is merged - this waits on adding the other HLP contributions into team2-wire!

**Comment by tomcl** [comment]
mostly - maybe all? fixed. close for now.
